AppLovin MAX Demo App - ObjC/AppLovin MAX Demo App - ObjC/ALAppDelegate.m

```diff
@@ -14,22 +14,22 @@ @implementation ALAppDelegate
 
 - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions
 {
-    // If you want to test your own AppLovin SDK key,
-    // update the value in Info.plist under the \"AppLovinSdkKey\" key, and update the package name to your app's name.
-    
-    // Enable test mode by default for the current device.
-    NSString *currentIDFV = UIDevice.currentDevice.identifierForVendor.UUIDString;
-    ALSdkSettings *settings = [[ALSdkSettings alloc] init];
-    
-    if ( currentIDFV.length > 0 )
-    {
-        settings.testDeviceAdvertisingIdentifiers = @[currentIDFV];
-    }
-    
-    // Initialize the AppLovin SDK
-    ALSdk *sdk = [ALSdk sharedWithSettings: settings];
-    sdk.mediationProvider = ALMediationProviderMAX;
-    [sdk initializeSdkWithCompletionHandler:^(ALSdkConfiguration *configuration) {
+    // Create the initialization configuration
+    // If you want to test your own AppLovin SDK key, update the value here and update the package name to your app's name.
+    ALSdkInitializationConfiguration *configuration = [ALSdkInitializationConfiguration configurationWithSdkKey: @"05TMDQ5tZabpXQ45_UTbmEGNUtVAzSTzT6KmWQc5_CuWdzccS4DCITZoL3yIWUG3bbq60QC_d4WF28tUC4gVTF" builderBlock:^(ALSdkInitializationConfigurationBuilder *builder) {
+
+        builder.mediationProvider = ALMediationProviderMAX;
+        
+        // Enable test mode by default for the current device.
+        NSString *currentIDFV = UIDevice.currentDevice.identifierForVendor.UUIDString;
+        if ( currentIDFV.length > 0 )
+        {
+            builder.testDeviceAdvertisingIdentifiers = @[currentIDFV];
+        }
+    }];
+
+    // Initialize the SDK with the configuration
+    [[ALSdk shared] initializeWithConfiguration: configuration completionHandler:^(ALSdkConfiguration *configuration) {
```

Comment:
This is confusing - I'd not have shadowing of the inner `configuration` object - can we name the outer one `initConfig` and inner `sdkConfig`?